### ISSUE TITLE
Google Music for Desktop don`t work on Kubuntu 18.10

### ISSUE THREAD

**Issue by ihusky02** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#3540]
**Checklist:**

* [X] I have checked that there are no issues with similar or the same content

  **YOU SHOULD CHECK CLOSED ISSUES ASWELL**
* [X] I have checked the FAQ (https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/wiki/FAQ) and the answer I am looking for is not there
* [X] I have double checked and can reproduce the issue

**OS:** Kubuntu 18.10

**GPMDP Version:**4.6.1

**Issue Descriptions:**
When I click on app, circle next to mouse cursor turn about five seconds and nothing more (on Linux Mint I don`t had this problem)
**Steps to Reproduce:**
Click on app

**Comment by heyraffaello** [comment]
Can confirm this issue in Ubuntu 19.04. Here's what I'm getting from syslog:

`kernel: [  672.773195] google-play-mus[9956]: segfault at a4fa30 ip 0000000000a4fa30 sp 00007fffc7577ff8 error 15 in Google Play Music Desktop Player[200000+1602000]
Apr 19 11:42:00 justin-laptop kernel: [  672.773200] Code: da 9d ef 9d e6 9d f2 9d f8 9d 0c 9e fa 9d 1b 9e 64 76 58 76 67 76 02 76 46 76 47 76 49 76 1e 76 3b 76 6d 76 6e 76 69 76 72 76 <c7> 7a b6 7a 60 89 33 89 5d 89 47 89 38 89 64 89 b8 76 2e 80 2c 80`

Hopefully this helps get the issue solved.

**Comment by heyraffaello** [comment]
Can confirm this issue in Ubuntu 19.04. Here's what I'm getting from syslog:

`xxxxx-laptop kernel: [  672.773195] google-play-mus[9956]: segfault at a4fa30 ip 0000000000a4fa30 sp 00007fffc7577ff8 error 15 in Google Play Music Desktop Player[200000+1602000]
Apr 19 11:42:00 xxxxx-laptop kernel: [  672.773200] Code: da 9d ef 9d e6 9d f2 9d f8 9d 0c 9e fa 9d 1b 9e 64 76 58 76 67 76 02 76 46 76 47 76 49 76 1e 76 3b 76 6d 76 6e 76 69 76 72 76 <c7> 7a b6 7a 60 89 33 89 5d 89 47 89 38 89 64 89 b8 76 2e 80 2c 80`

Hopefully this helps get the issue solved.

**Comment by axelitus** [comment]
Same here, can confirm I get segmentation fault in Ubuntu 19.04 fresh install.

**Comment by olealgoritme** [comment]
Yeap, ubuntu 19.04 segfaults with current official release. I suggest cloning this github repo, npm build && npm start

**Comment by MarshallOfSound** [comment]
This is a known issue, a new release might be on it's way out soon

**Comment by pauldotknopf** [comment]
For the sake of information, it is also crashing on fresh Ubuntu 18.10.

```
gdb /usr/bin/google-play-music-desktop-player                                                                           SIGSEGV(11) ↵  10142  08:32:00 
GNU gdb (Ubuntu 8.2-0ubuntu1) 8.2
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Type "show copying" and "show warranty" for details.
This GDB was configured as "x86_64-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
    <http://www.gnu.org/software/gdb/documentation/>.

For help, type "help".
Type "apropos word" to search for commands related to "word"...
Reading symbols from /usr/bin/google-play-music-desktop-player...(no debugging symbols found)...done.
(gdb) run
Starting program: /usr/bin/google-play-music-desktop-player 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".

Program received signal SIGSEGV, Segmentation fault.
0x0000000000a4fa30 in ?? ()
(gdb) backtrace
#0  0x0000000000a4fa30 in  ()
#1  0x00007ffff7d832b2 in node::http2::Http2Session::Callbacks::Callbacks(bool) () at /usr/share/google-play-music-desktop-player/libnode.so
#2  0x00007ffff7d83375 in  () at /usr/share/google-play-music-desktop-player/libnode.so
#3  0x00007ffff7fe398a in call_init (l=<optimized out>, argc=argc@entry=1, argv=argv@entry=0x7fffffffde48, env=env@entry=0x7fffffffde58) at dl-init.c:72
#4  0x00007ffff7fe3a89 in call_init (env=0x7fffffffde58, argv=0x7fffffffde48, argc=1, l=<optimized out>) at dl-init.c:30
#5  0x00007ffff7fe3a89 in _dl_init (main_map=0x7ffff7ffe190, argc=1, argv=0x7fffffffde48, env=0x7fffffffde58) at dl-init.c:119
#6  0x00007ffff7fd40ca in _dl_start_user () at /lib64/ld-linux-x86-64.so.2
#7  0x0000000000000001 in  ()
#8  0x00007fffffffe1cb in  ()
#9  0x0000000000000000 in  ()
(gdb)
```

**Comment by heyraffaello** [comment]
This minor release fixes the issue: https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/issues/3548

**Comment by pauldotknopf** [comment]
I just confirmed that this release indeed did fix my issue on Ubuntu 18.10.

Thanks!
